include/sketch.h

```diff
@@ -12,14 +12,14 @@ using namespace std;
  */
 class Sketch {
   const long seed;
-  const int n;
+  const unsigned long long int n;
```

Comment:
unsigned long long is fine (and correct) but consider using uint64_t in the future. I think it shows intent more clearly. other integer types in C++ aren't as well defined as long longs (which are guaranteed to be at least 64 bits, but not necessarily exactly 64 bits)

Reply:
good point, I'll change that with the other changes in the PR